docs/current_docs/manuals/developer/snippets/secrets/secret-variable/typescript/index.ts

```diff
@@ -0,0 +1,16 @@
+import { dag, object, func, Secret } from "@dagger.io/dagger"
+
+@object()
+class MyModule {
+  @func()
+  async githubAauth(secret: Secret): Promise<string> {
+    return await dag
+      .container()
+      .from("alpine:3.17")
+      .withExec(["apk", "add", "github-cli"])
+      .withMountedSecret("/root/.config/gh/hosts.yml", secret)
```

Comment:
same question here, shouldn't it be `withSecretVariable`?

Reply:
yup, I had these right before and then I think I copied the content from the secret-file snippets over top of these